asm/istio/istio-operator.yaml

```diff
@@ -14,14 +14,10 @@
 
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
```

Comment:
We've simplified the configs to reduce some input from customers. We can just copy the file below:
https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/release-1.7-asm/asm/istio/istio-operator.yaml

Then we can remove those 3 kpt commands below from your install script:
```
kpt cfg set xxx/asm gcloud.compute.location "$ZONE"
kpt cfg set xxx/asm gcloud.project.projectNumber "$PROJECT_NUMBER"
kpt cfg set xxx/asm gcloud.container.cluster "$CLUSTER"
```